students/ChengLiu/session04/trigrams.py

```diff
@@ -15,6 +15,9 @@
     with open(filename, 'r'):
         lines = file.read()
         words = lines.split()
+    for i in words:
+        if i in ('-', ',', '.'):
+            i = ''
```

Comment:
as a rule, when you want to replace characters, you want to use str.replace() -- or str.translate()